### PR TITLE
Docs for toUnixTimestamp

### PR DESCRIPTION
**Pull request by godfreyd**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation

### CODE REVIEW

File: docs/en/query_language/functions/date_time_functions.md [hunk 1]
```diff
@@ -62,9 +62,42 @@ Converts a date with time to a UInt8 number containing the number of the minute
...
+
+**Syntax** 
+
+```sql
+toUnixTimestamp(datetime[, timezone])
+```
+
+**Parameters**
+
+- `datetime` — Date with time. [DateTime](../../data_types/datetime.md#data_type-datetime).
```
Comment (alexey-milovidov) [inline docs/en/query_language/functions/date_time_functions.md]: Wrong, see example.

File: docs/en/query_language/functions/date_time_functions.md [hunk 2]
```diff
@@ -62,9 +62,42 @@ Converts a date with time to a UInt8 number containing the number of the minute
...
+- Returns the number of seconds between the specified date and the Unix epoch (1970-01-01 00:00:00 UTC).
+
+Type: `UInt32`.
+
+**Example**
+
+Query:
+
+```sql
+SELECT toUnixTimestamp('2017-11-05 08:07:47', 'Asia/Tokyo') AS unix_timestamp
```
Comment (alexey-milovidov) [inline docs/en/query_language/functions/date_time_functions.md]: Data type in this example is String, not DateTime.

File: docs/en/query_language/functions/date_time_functions.md [hunk 3]
```diff
@@ -62,9 +62,42 @@ Converts a date with time to a UInt8 number containing the number of the minute
...
+**Syntax** 
+
+```sql
+toUnixTimestamp(datetime[, timezone])
+```
+
+**Parameters**
+
+- `datetime` — Date with time. [DateTime](../../data_types/datetime.md#data_type-datetime).
+- `timezone` — Optional time zone argument. [String](../../data_types/string.md).
```
Comment (alexey-milovidov) [inline docs/en/query_language/functions/date_time_functions.md]: Timezone parameter does not make sense if DateTime value is passed in first parameter.

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
.

File: docs/en/query_language/functions/date_time_functions.md (hunk 2 above)
Comment (godfreyd) [reply docs/en/query_language/functions/date_time_functions.md]: Fix it

File: docs/en/query_language/functions/date_time_functions.md (hunk 3 above)
Comment (godfreyd) [reply docs/en/query_language/functions/date_time_functions.md]: Fix it

File: docs/en/query_language/functions/date_time_functions.md (hunk 1 above)
Comment (godfreyd) [reply docs/en/query_language/functions/date_time_functions.md]: Fix it

File: docs/en/query_language/functions/date_time_functions.md [hunk 4]
```diff
@@ -62,9 +62,42 @@ Converts a date with time to a UInt8 number containing the number of the minute
...
+
+**Syntax** 
+
+```sql
+toUnixTimestamp(datetime[, timezone])
+```
+
+**Parameters**
+
+- `datetime` — Date with time. [String](../../data_types/string.md).
```
Comment (alexey-milovidov) [inline docs/en/query_language/functions/date_time_functions.md]: It may have either String or DateTime data type.

Comment (alexey-milovidov) [reply docs/en/query_language/functions/date_time_functions.md]: Timezone argument makes sense only if the datetime argument was in String form.

File: docs/ru/query_language/functions/date_time_functions.md [hunk 5]
```diff
@@ -43,6 +43,43 @@ SELECT
...
+```
+
+**Параметры**
+
+- `datetime` — Дата с временем. [String](../../data_types/string.md).
+- `timezone` — Опциональный параметр. Часовой пояс. [String](../../data_types/string.md).
+
+**Возвращаемое значение**
+
+- Возвращает количество секунд, прошедших с полуночи (00:00:00 UTC) 1 января 1970 года и указанной датой.
```
Comment (alexey-milovidov) [inline docs/ru/query_language/functions/date_time_functions.md]: И указанным временем.

Comment (alexey-milovidov) [inline docs/ru/query_language/functions/date_time_functions.md]: Без учёта секунд координации.

**Pull request merged**